config/routes.rb

```diff
@@ -222,6 +222,7 @@
   put 'moderate/spam/:id' => 'admin#mark_spam'
   put 'moderate/publish/:id' => 'admin#publish'
   put 'admin/promote/moderator/:id' => 'admin#promote_moderator'
+  get 'resetuserpassword/:id' => 'admin#reset_user_password'
```

Comment:
I think this name might not be that great and another path name would be good.

Reply:
maybe... `/user/reset/password` ?